tests/basic/basic_test.go

```diff
@@ -295,9 +298,6 @@ func getMaxNodesToBeDown(replicas int) int {
 	if replicas == 1 {
 		return 0
 	}
-	if replicas%2 != 0 {
-		return replicas/2 + 1
-	}
 	return replicas / 2
```

Comment:
if number of replicas is 3, it is okay to shutdown 2 PX nodes in a 5 node cluster. With this change, it will only shutdown 1 node in a 5 node cluster.